package/MDAnalysis/coordinates/TRC.py

```diff
@@ -224,129 +224,141 @@ def _read_traj_properties(self):
         l_timestep_timevalues = []
 
         with util.anyopen(self.filename) as f:
-            for line in iter(f.readline, ''):
-
+            for line in iter(f.readline, ""):
                 #
                 # First block of frame
                 #
-                if (first_block == line.strip()):
-                    l_blockstart_offset.append(f.tell()-len(line))
+                if first_block == line.strip():
+                    l_blockstart_offset.append(f.tell() - len(line))
                     frame_counter += 1
 
                 #
                 # Timestep-Block
                 #
-                if ("TIMESTEP" == line.strip()):
+                if "TIMESTEP" == line.strip():
                     lastline_was_timestep = True
 
-                elif (lastline_was_timestep is True):
+                elif lastline_was_timestep is True:
                     l_timestep_timevalues.append(float(line.split()[1]))
                     lastline_was_timestep = False
 
                 #
                 # Coordinates-Block
                 #
-                if ("POSITIONRED" == line.strip()):
+                if "POSITIONRED" == line.strip():
                     in_positionred_block = True
 
                 if (in_positionred_block is True) and (n_atoms == 0):
-                    if (len(line.split()) == 3):
+                    if len(line.split()) == 3:
                         atom_counter += 1
 
                 if ("END" == line.strip()) and (in_positionred_block is True):
                     n_atoms = atom_counter
                     in_positionred_block = False
 
+        if frame_counter == 0:
+            raise ValueError(
```

Comment:
Is this likely to happen? It sounds like something that ought to be tested.

Reply:
Yes, this should trigger if the trajectory file is empty.
I added a test.